utils/standalone_searx.py

```diff
@@ -1,5 +1,74 @@
 #!/usr/bin/env python
+"""Script to run searx from terminal.
 
+Import searx directly will raise error.
+User have to set `SEARX_DEBUG` env or searx's secret key to pass that error
+
+>>> import searx
+ERROR:searx:server.secret_key is not changed. Please use something else instead of ultrasecretkey.
+NameError: name 'exit' is not defined
+>>> from os import environ
+... environ.setdefault('SEARX_DEBUG', 'true')
+... import searx
```

Comment:
Should we handle the SEARX_DEBUG error in the docstring? .. IMO it is a bug (https://github.com/searx/searx/issues/2278) which is partial solved in [PR 2287](https://github.com/searx/searx/pull/2287).

Reply:
so it will be guaranteed that searx can be imported without changing everything?